app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,43 @@
+@import "design-patterns/breadcrumbs";
+@import "mixins/back-arrow";
+@import "mixins/media-down";
+@import "mixins/touch-friendly-links";
+
+.govuk-breadcrumbs {
```

Comment:
As we're moving the component into the gem,  we should change it to meet our component principles. One of these is we need to rename the namespaced CSS classes, e.g: this would be `gem-c-breadcrumbs`. More info here: https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md#namespacing

Reply:
All the classes need to match the `gem-c-` pattern